docs/optimization_guide/model_optimization_guide.md

```diff
@@ -27,9 +27,13 @@
 
 To understand which development optimization tool you need, refer to the diagram: 
 
-POT is the easiest way to get optimized models, and usually takes several minutes depending on the model size and used HW. NNCF can be considered as an alternative or addition when the first one does not give accurate results. 
+Post-training methods are limited in terms of achievable accuracy and for challenging usecases accuracy might degrade. In this case, training-time optimization with NNCF is an option.
+
+Once the model is optimized using the aforementioned tools it can be used for inference using the regular OpenVINO inference workflow. No changes of the code are required.
```

Comment:
```suggestion
Once the model is optimized using the aforementioned tools it can be used for inference using the regular OpenVINO inference workflow. No changes to the code are required.
```